### PR TITLE
update instructions

### PR DESCRIPTION
**Pull request by kchia**
@dalelotts Since the students only meet with the mentor once or twice a week and rarely do mentors interact with the students outside of the face-to-face meetings, I'm not sure if mentors would be able to give them timely help.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,20 @@
 # Starter Code: Robust server structure
 
-This server is intended to be run for some checkpoints in the Thinkful curriculum. If you have trouble getting the server to run, reach out to your mentor.
+This server is intended to be run for some checkpoints in the Thinkful curriculum. If you have trouble getting the server to run, reach out for assistance.
```
Comment (kchia) [inline README.md:3]: Keeping this generic so they can reach out to whatever mode of assistance Thinkful currently offers (e.g., Slack channels)

File: README.md [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
 
-## Installation Instructions
+## Installation
 
 1. Clone this repository.
 1. `cd` into the newly created directory.
 1. Run `npm install`.
-1. Run `npm start`.
```
Comment (kchia) [inline README.md:10]: To keep things simple, why don't we just steer the student towards one way of running their server for this exercise.

Comment (dalelotts) [reply README.md:10]: I'm not sure of a reasonable approach to resolve differences in opinion like this. Do you have a suggestion? 

`npm start` is the standard way to run a node project. If they go to an interview and do not know  npm start that will be a problem. Additionally, I think if they don't know that they can create different run scripts to add behavior that seems like a problem as well. To your point, the focus if this lesson is not the run scripts so maybe we don't need the standard way here. So I guess the open questions I have are:

- Do we agree that the students need to know `npm start` _and_ how to create other start scripts?
- If yes to the above, where should this information be included in the curriculum?

Comment (kchia) [reply README.md:10]: Great point! `npm start` was covered in module 3.1.

**Pull request merged**